openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -84,6 +84,9 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
             % endif
         </div>
         <aside class="page-content-secondary course-sidebar">
+            % if show_proctoring_info_panel:
+                <div class="proctor-test" data-course-id="${course_key}"></div>
```

Comment:
NIT: think this class name is left over from testing the POC, can we use a more descriptive class name?

Reply:
Yep!